apps/sms/views/conversation/js/attachment_renderer.js

```diff
@@ -203,6 +203,16 @@
 
               // It's essential to remember real image URL to revoke it later
               attachmentContainer.dataset.thumbnail = data.thumbnail.url;
+
+              var fileNode = contentContainer.querySelector('.file-name');
+              if (this._attachment.name) {
+                fileNode.removeAttribute('data-l10n-id');
+                fileNode.textContent = this._attachment.name.slice(
+                  this._attachment.name.lastIndexOf('/') + 1
```

Comment:
tell me what you think, but I also wonder if we couldn't use formatValue in getBaseMarkup to get the localized value ? getBaseMarkup would then return a Promise. So we wouldn't need to do some DOM operation after rendering.

what do you think ?